2006.1.1

desht opened the pull request:
Bump arch dep version for NeoForge compat

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=91) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=91) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/FTBTeam/FTB-Library-Legacy?pullRequest=91) it.</sub>

desht committed:
fix: comment out usage of Architectury JEI fluid ingredient type

Current code crashes because the FLUID_STACK type isn't registered with JEI
The type needs to be registered, along with a helper and renderer and this
doesn't feel worth it, given that we already have working support for native
fluid types on all three platforms.

desht committed:
Merge remote-tracking branch 'origin/dev' into dev

desht committed:
build: use arch 12.1.3 for compat with latest neoforge

desht committed:
build: version -> 2006.1.1
